docs/source/working_with_aiida/troubleshooting.rst

```diff
@@ -15,32 +15,42 @@ Only warnings and errors are logged to the daemon log files, while info and debu
 messages are discarded.
 
 If you are experiencing a problem, you can change the default minimum logging
-level of AiiDA messages (and circus messages -- circus is the library that we
-use to manage the daemon process) using, on the command line, the two
-following commands::
+level of AiiDA messages::
 
-  verdi config logging.circus_loglevel DEBUG
   verdi config logging.aiida_loglevel DEBUG
 
+You might also be interested in circus log messages (the ``circus`` library is the daemonizer that manages the daemon runners) but most often it is used by AiiDA developers::
+
+  verdi config logging.circus_loglevel DEBUG
+
+
 For each profile that runs a daemon, there will be two unique logfiles, one for
-AiiDA log messages and one from the circus daemonizer. These files can be found
-in the ``.aiida`` log folder, which by default can be found at ``~/.aiida/daemon/log/aiida_daemon.log``.
+AiiDA log messages (named ``aiida-<profile_name>.log``) and one for the circus logs (named ``circus-<profile_name>.log``). Those files can be found
+in the ``~/.aiida/daemon/log`` folder.
+
 After rebooting the daemon (``verdi daemon restart``), the number of messages
 logged will increase significantly and may help in understanding
 the source of the problem. 
 
 .. note:: In the command above, you can use a different level than ``DEBUG``.
   The list of the levels and their order is the same of the `standard python
-  logging module <https://docs.python.org/2/library/logging.html#logging-levels>`_.
+  logging module <https://docs.python.org/3/library/logging.html#logging-levels>`_.
+  In addition to the standard loggin levels we define our custom ``REPORT`` level
+  that has a value of ``23``.
```

Comment:
Let's make this sentence:

In addition to the standard logging levels, we define our custom ``REPORT`` level, which, with a value of ``23``, sits between the standard ``INFO`` and ``WARNING`` levels. The ``REPORT`` level is the default logging level as this is what is used by messages from, among other things, the work chain report.

Reply:
done